features.py

```diff
@@ -0,0 +1,104 @@
+"""
+Script to compute audio features for the audio recordings in the Eisenberg Collection.
+The script goes through the song titles in the metadata file and checks if there are
+audio file paths that contain the title. If there is a match, the features are computed
+and added to a dictionary with the NYUAD Archive reference as a key. Audio file paths
+that have been matched are removed from the list of paths, so that the audio recordings
+that haven't been matched are provided to the user at the end.
+"""
+
+import librosa #feature computation
+from pathlib import Path #directory traversal
+import pandas as pd #read metadata DataFrame
+import argparse #parse command line arguments
+import deepdish as dd #save python dictionary to HDF5
+import sys #display computation progress
+import warnings #to filter warnings
+from tables import NaturalNameWarning #to filter naming warning from PyTables
+
+# Argument parser
+parser = argparse.ArgumentParser(description="audio and metadata directories")
+parser.add_argument("--metadata_path", required=True)
+parser.add_argument("--audio_dir", default='.')
+parser.add_argument("--output_dir", default='.')
+args = parser.parse_args()
+metadata_path = args.metadata_path
+audio_dir = args.audio_dir
+output_dir = args.output_dir
+
+# filter naming warnings from PyTables
+warnings.filterwarnings('ignore', category=NaturalNameWarning)
```

Comment:
added to issue #6

Reply:
What is the issue with having warnings? Messing up the prints?

We may want to keep the warnings visible for future compatibility reasons.

Reply:
Thanks for the [explanation](https://github.com/chrispla/Eisenberg_Collection_helper/pull/5#issuecomment-867366295). Can you put a reference to the StackOverflow thread for future developers, e.g.

```
# filter redundant naming warnings from PyTables. We do not use PyTables. See:
# https://stackoverflow.com/questions/58414068/how-to-get-rid-of-naturalnamewarning
```

Reply:
Thanks, explanation added to issue #6 .

Reply:
Sorry, I misunderstood! I added an explanation on the script too on commit 3fb2ff2388e4f992593a96a693ee07ee7170987f.